mobilecoind-json/src/main.rs

```diff
@@ -0,0 +1,317 @@
+#![feature(proc_macro_hygiene, decl_macro)]
+
+use grpcio::{ChannelBuilder, ChannelCredentialsBuilder};
+use protobuf::RepeatedField;
+
+use mc_api::external::KeyImage;
+use mc_common::logger::{create_app_logger, log, o};
+use mc_mobilecoind_api::mobilecoind_api_grpc::MobilecoindApiClient;
+use rocket::{get, post, routes};
+use rocket_contrib::json::Json;
+use serde_derive::{Deserialize, Serialize};
+use std::sync::Arc;
+use structopt::StructOpt;
+
+/// Command line config, set with defaults that will work with
+/// a standard mobilecoind instance
+#[derive(Clone, Debug, StructOpt)]
+#[structopt(
+    name = "mobilecoind-rest-gateway",
```

Comment:
`mobilecoind-json` instead of `mobilecoind-rest-gateway`?

Reply:
I can rename after doing the other stuff so I don't lose your other comments

Reply:
This still needs to be renamed to reflect the current name of the crate.